src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -14,6 +14,7 @@ import {
 
 export interface ModalManagerProps {
   submitUrls?: { [P in ModalType]: string } & {
+    apple?: string
```

Comment:
Although the existing submit urls are empty, I have a feeling we will want to make them required-- our auth forms will not work without them. If you are able to make the apple types required, I'm happy to follow up on facebook and removing deprecated twitter auth.